### ISSUE TITLE
Crash at launch

### ISSUE THREAD

**Issue by xperiandri** [TeamCodeStream/CodeStream#50]
``` C#
Microsoft.VisualStudio.Composition.CompositionFailedException: This part (CodeStream.VisualStudio.UI.TextViewCreationListener) cannot be instantiated.
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.ThrowPartNotInstantiableException()
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.GetValueReadyToExpose()
at Microsoft.VisualStudio.Composition.ExportProvider.<>c__DisplayClass53_0.<CreateExport>b__0()
at System.Lazy`1.CreateValue()at System.Lazy`1.LazyInitValue()at System.Lazy`1.get_Value()
at Microsoft.VisualStudio.Composition.NetFxAdapters.MefV1ExportProvider.<>c__DisplayClass10_0.<UnwrapExport>b__1()
at System.ComponentModel.Composition.Primitives.Export.GetExportedValueCore()
at System.ComponentModel.Composition.Primitives.Export.get_Value()
at System.ComponentModel.Composition.ExportServices.GetCastedExportedValue[T](Export export)
at System.ComponentModel.Composition.ExportServices.<>c__DisplayClass11_0`2.<CreateStronglyTypedLazyOfTM>b__1()at System.Lazy`1.CreateValue()at System.Lazy`1.LazyInitValue()
at System.Lazy`1.get_Value()
at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.InstantiateExtension[TExtension,TMetadata](Object errorSource, Lazy`2 provider)
 --- End of stack trace from previous location where exception was thrown ---
at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

**Comment by bcanzanella** [comment]
Hi @xperiandri can you post the version details for Visual Studio? From Visual Studio, go to `Help -> About Microsoft Visual Studio` click the `Copy Info` button and paste here the first part, it will look like this:

```
Microsoft Visual Studio Community 2019 RC
Version 16.0.0
VisualStudio.16.Release/16.0.0+28729.10
Microsoft .NET Framework
Version 4.7.03190

Installed Version: Community
```

thanks!

**Comment by xperiandri** [comment]
```
Microsoft Visual Studio Enterprise 2019
Version 16.0.1
VisualStudio.16.Release/16.0.1+28803.156
Microsoft .NET Framework
Version 4.7.03190

Installed Version: Enterprise

Architecture Diagrams and Analysis Tools   00435-60000-00000-AA759
Microsoft Architecture Diagrams and Analysis Tools

Visual C++ 2019   00435-60000-00000-AA759
Microsoft Visual C++ 2019

.ignore   1.2.77
A language service that makes it painless to handle all types of .ignore files such as .gitignore, .tfignore, etc.

ADL Tools Service Provider   1.0
This package contains services used by Data Lake tools

ASP.NET and Web Tools 2019   16.0.12311.10635
ASP.NET and Web Tools 2019

ASP.NET Core Blazor Language Services   16.0.19154.02

ASP.NET Web Frameworks and Tools 2019   16.0.12311.10635
For additional information, visit https://www.asp.net/

AvaloniaPackage Extension   1.0
AvaloniaPackage Visual Studio Extension Detailed Info

Azure App Service Tools v3.0.0   16.0.12311.10635
Azure App Service Tools v3.0.0

Azure Data Lake Node   1.0
This package contains the Data Lake integration nodes for Server Explorer.

Azure Data Lake Tools for Visual Studio   2.3.8000.0
Microsoft Azure Data Lake Tools for Visual Studio

Azure Functions and Web Jobs Tools   16.0.12311.10635
Azure Functions and Web Jobs Tools

Azure Stream Analytics Tools for Visual Studio   2.3.8000.0
Microsoft Azure Stream Analytics Tools for Visual Studio

C# Tools   3.0.0-beta4-19170-01+1deafee3682da88bf07d1c18521a99f47446cee8
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Clear MEF Component Cache   1.4
Clears the Visual Studio MEF component cache to fix issues with cache corruption.

CodeRush for Roslyn   18.2.9.0
DevExpress CodeRush for Roslyn package.

CodeRush for Roslyn Tool Windows   18.2.9.0
DevExpress CodeRush for Roslyn Tool Windows package.

CodeStream   0.52.0.381
Discuss, review, and share code with your team. Integrates w/ Slack, Jira, Trello, and Live Share

Command Task Runner   1.2.41
Adds support for command line batch files in Visual Studio 2015's Task Runner Explorer. Supports .cmd, .bat, .ps1 and .psm1 files.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

CSS AutoPrefixer   0.7.10
Write your CSS rules without vendor prefixes (inÂ fact, forget about them entirely). Autoprefixer will use the data based on current browser popularity and property support toÂ apply prefixes forÂ you.

CSS Sorter   0.8.7
Sort CSS properties easily for better readibility and GZIP compression

EditorConfig Language Service   1.17.258
Language service for .editorconfig files.

EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs.

EditProj   1.0
This packages allows you to edit project and solution files without the need to unload them first.

Extensibility Message Bus   1.1.77 (master@24013d5)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

Fabric.DiagnosticEvents   1.0
Fabric Diagnostic Events

File Differ   1.4.25
The easiest way to diff two files directly in Solution Explorer

File Icons   2.7
Adds icons for files that are not recognized by Solution Explorer

Fix File Encoding   1.4.1
Prevents Visual Studio from adding BOM to UTF-8 files.

Generate GUID for Visual Studio 2012   1.0
Generate a variety of GUID values with a click or by assigning a chord to the menu command with the ability to change formats using a drop down in the Tools menu.

Gist Manager   1.0
Gist Manager lets you manage your Github gists.

GitBlamerCommand Extension   1.0
GitBlamerCommand Visual Studio Extension Detailed Info

GitHub.VisualStudio   2.9.0.7614
A Visual Studio Extension that brings the GitHub Flow into Visual Studio.

IntelliCode Extension   1.0
IntelliCode Visual Studio Extension Detailed Info

IntelliSenseExtenderPackage Extension   1.0
IntelliSenseExtenderPackage Visual Studio Extension Detailed Info

Markdown Editor   1.12.236
A full featured Markdown editor with live preview and syntax highlighting. Supports GitHub flavored Markdown.

Menu Extension   1.0
Menu Visual Studio Extension Detailed Info

Microsoft Azure HDInsight Azure Node   2.3.8000.0
HDInsight Node under Azure Node

Microsoft Azure Hive Query Language Service   2.3.8000.0
Language service for Hive query

Microsoft Azure Service Fabric Tools for Visual Studio   2.5
Microsoft Azure Service Fabric Tools for Visual Studio

Microsoft Azure Stream Analytics Language Service   2.3.8000.0
Language service for Azure Stream Analytics

Microsoft Azure Stream Analytics Node   1.0
Azure Stream Analytics Node under Azure Node

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 0x10 - v2.9.0.0

Microsoft Continuous Delivery Tools for Visual Studio   0.4
Simplifying the configuration of Azure DevOps pipelines from within the Visual Studio IDE.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft Library Manager   1.0
Install client-side libraries easily to any web project

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual C++ Wizards   1.0
Microsoft Visual C++ Wizards

Microsoft Visual Studio Tools for Containers   1.1
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

Mono Debugging for Visual Studio   16.0.300 (573eda3)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   5.0.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Open Command Line   2.1.218
Opens a command line at the root of the project. Support for all consoles such as CMD, PowerShell, Bash etc. Provides syntax highlighting, Intellisense and execution of .cmd and .bat files.

Open in NSwagStudio   1.1
Adds a menu command that lets you open nswag file in NSwagStudio.

Package Installer   2.0.103
Makes it easier, faster and more convenient than ever to install Bower, npm, Yarn, JSPM, TSD, Typings and NuGet packages to any project

Package Security Alerts   1.0.22
Identifies npm and Bower packages that contains known security vulnerabilities to ensure your project is always using the most secure package versions.

PowerShell Pro Tools for Visual Studio   1.0
A set of tools for developing and debugging PowerShell scripts and modules in Visual Studio.

Project File Tools   1.0.1
Provides Intellisense and other tooling for XML based project files such as .csproj and .vbproj files.

Project System Tools   1.0
Tools for working with C#, VisualBasic, and F# projects.

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

ReviewsToolWindow Extension   1.0
ReviewsToolWindow Visual Studio Extension Detailed Info

Snapshot Debugging Extension   1.0
Snapshot Debugging Visual Studio Extension Detailed Info

SQL Server Data Tools   16.0.61903.25110
Microsoft SQL Server Data Tools

SteroidsVS   1.0
Multi-purpose extension for VisualStudio

StylerPackage Extension   1.0
StylerPackage Visual Stuido Extension Detailed Info

Surface Dial Tools for Visual Studio   0.8.68
Adds features to Visual Studio specific to the Surface Dial. You don't need a Surface Book or Surface Studio to take advantage of the Surface Dial.

ToastNotifier   1.0
Show Toast Notification when Bulid completed.

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TrimCopyPackage Extension   1.0
TrimCopyPackage Visual Studio Extension Detailed Info

TypeScript Tools   16.0.10306.2001
TypeScript Tools for Microsoft Visual Studio

uCalc Search/Transform   1.0
Search through code in innovative ways. Instantly transform or auto-generate code as well. For more info, visit: https://www.ucalc.com

Visual Basic Tools   3.0.0-beta4-19170-01+1deafee3682da88bf07d1c18521a99f47446cee8
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.4 for F# 4.6   16.0.0.0.  Commit Hash: 809f41207b74a1356208f11b8fa3a15233734050.
Microsoft Visual F# Tools 10.4 for F# 4.6

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for Containers   1.0
Visual Studio Tools for Containers

Visual Studio Tools for Kubernetes   1.0
Visual Studio Tools for Kubernetes

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

WebPack Task Runner   1.5.94
Adds support for WebPack in Visual Studio 2015 and 2017's Task Runner Explorer.

Windows Template Studio   3.1.19092.1
Windows Template Studio quickly builds a UWP app, using a wizard-based UI to turn your needs into a foundation of Windows 10 patterns and best practices.

Xamarin   16.0.0.515 (d16-0@a3502d929)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   4.17.4.418 (remotes/origin/d16-0@3d086e814)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   16.1.47 (31f664d)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   9.2.0.5 (HEAD/292d27fa)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: mono/mono/2018-08-rc@5ad371dab1b
    Java.Interop: xamarin/java.interop/d16-0@c987483
    LibZipSharp: grendello/LibZipSharp/master@44de300
    LibZip: nih-at/libzip/rel-1-5-1@b95cf3f
    MXE: xamarin/mxe/xamarin@b9cbb535
    ProGuard: xamarin/proguard/master@905836d
    SQLite: xamarin/sqlite/3.26.0@325e91a
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-0@0a7edd6


Xamarin.iOS and Xamarin.Mac SDK   12.8.0.0 (0aa8452)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

Yarn Installer   0.5.3
Makes it easy to restore npm packages using Yarn from within Visual Studio

ZenCoding   1.2.12
Provides ZenCoding for the HTML Editor - full support for static HTML, Razor and WebForms.
```

**Comment by bcanzanella** [comment]
thanks @xperiandri appreciate it. one more... could you post the contents of Visual Studio's MEF log? 
You can find this at: `%localappdata%\Microsoft\VisualStudio\16.0_<<RandomString>>\ComponentModelCache\Microsoft.VisualStudio.Default.err` (Replacing `<<RandomString>>` with what is actually your path). 

thanks for your help!
